src/csharp/Microsoft.Spark.Extensions.Delta.E2ETest/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
can we put extension projects under extensions folder?

Reply:
Moved